### PR TITLE
Libreehr rel200

### PR DESCRIPTION
**Pull request by tmccormi**
Release Candidate 1 - Review and Test. 

Update: it will be merged and THEN I'll create a release branch ...

### CODE REVIEW

**Comment by aethelwulffe** [comment]
If you install the mips stuff, it drops the original table and installs the new anyway.

**Comment by aethelwulffe** [comment]
Actually, we need the mips installer.

**Comment by tmccormi** [comment]
Should we just leave the MIPS as TBD patch release and turn off the menu in the meantime?

**Comment by tmccormi** [comment]
@aethelwulffe @teryhill give this one more look over and let me know if you guys are ready to release.

File: INSTALL.md [hunk 1]
```diff
@@ -164,49 +160,70 @@ Should anything fail during Step 4, you may have to remove the existing database
...
 
 Restart apache service. Instructions on doing that are given in the FAQ section.
 
 
 #### Step 6
-You will be given instructions on configuring the Apache web server.  We suggest you print these instructions for future reference. Instructions are given to secure the`libreehrwebroot/sites/*/documents`, `libreehrwebroot/sites/*/edi` and `libreehrwebroot/sites/*/era` directories, which contain patient information. This can be done be either placing pertinent `.htaccess` files in these directories or by editing the apache configuration file. 
+You will be given instructions on configuring the Apache web server.  We suggest you print these instructions for future reference. Instructions are given to secure the`librehealthehrwebroot/sites/*/documents`, `librehealthehrwebroot/sites/*/edi` and `librehealthehrwebroot/sites/*/era` directories, which contain patient information. This can be done be either placing pertinent `.htaccess` files in these directories or by editing the apache configuration file.
```
Comment (aethelwulffe) [inline INSTALL.md]: Defining the webroot may be helpful.  LibreHealth_EHR_root_directory_location

**Comment by aethelwulffe** [comment]
We need nother round from you Terry.  Pretty sure we don't need to update clinical_rules.  MIPS does the whole deal.

**Comment by teryhill** [comment]
@tmccormi did you increment the version in the upgrade script?

**Comment by tmccormi** [comment]
Yes .. 
daad2a2 Release 2.0.0 RC1
M       INSTALL.md
M       README.md
M       setup.php
D       sql/1_0_1-to-1_0_2_upgrade.sql
A       sql/1_0_1-to-2_0_0_upgrade.sql

Although I can't recall if the $v_database version number should be "2" or '2.0.0'

**Comment by aethelwulffe** [comment]
Think the DB version is an integer increment.

**Comment by aethelwulffe** [comment]
A three-digit version should be fine...

**Pull request merged**